tests/tpchbench/run_ve.sh

```diff
@@ -27,7 +27,7 @@ time $SPARK_HOME/bin/spark-submit \
     --conf spark.cyclone.ve.columnBatchSize=128000 \
     --conf spark.cyclone.ve.targetBatchSizeMb=32 \
     --conf spark.sql.inMemoryColumnarStorage.batchSize=128000 \
-    --conf spark.sql.autoBroadcastJoinThreshold=-1 \
```

Comment:
Removing the `\`?

Reply:
Ah, that's an odd one.. Don't know how that got there.